import.js

```diff
@@ -0,0 +1,121 @@
+function importArticle() {
+  const codeN = document.getElementById("codeN");
+  const amount = document.getElementById("amountN");
+  const buy = document.getElementById("amountBtnN");
+  const imageN = document.getElementById("imangeN");
+  const billN = document.getElementById("billN");
+  const infoN = document.getElementById("importInfo");
+  const traziBtn = document.getElementById("searchN");
+
+  let komN = [[]];
+  let singleItemN = 0;
+
+  importDiv.style.display = "block";
+  codeN.focus();
+
+  imageN.addEventListener("click", () => {
+    if (komN.length > 1) {
+      komN[0] = new Date();
+      komN.push(singleItemN.toString());
+      let singleItem = [];
+      if (localStorage.getItem("nabavka")) {
+        let temp = JSON.parse(localStorage.getItem("nabavka"));
+        temp = temp.concat(komN);
+        localStorage.setItem("nabavka", JSON.stringify(temp));
+      } else localStorage.setItem("nabavka", JSON.stringify(komN));
+      for (let i = 1; i < komN.length - 1; i++) {
+        singleItem = storage[komN[i][0]];
+        singleItem.stock += komN[i][1];
+        singleItem.realStock += komN[i][1];
```

Comment:
Primer kako je lakse update-ovati property-e `Baze` umesto csv string-a.

Reply:
da. razmisljao sam da sve to sredim kad nadjem vremena, mada mi se vise svidja dvodimenzionalni niz od niza objekata